CHANGELOG.md

```diff
@@ -0,0 +1,42 @@
+## CHANGELOG
+
+### X.Y.Z (YYYY-mm-dd)
```

Comment:
This should be removed until the 1.2.0 release is complete.

Reply:
This has been removed.